sdk/eventhub/Azure.Messaging.EventHubs/src/Errors/EventHubsObjectClosedException.cs

```diff
@@ -0,0 +1,41 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+
+namespace Azure.Messaging.EventHubs.Errors
+{
+    /// <summary>
+    ///   An exception which occurs when an operation has been attempted using an Event Hubs
+    ///   object which has already been closed.
+    /// </summary>
+    ///
+    public sealed class EventHubsObjectClosedException : EventHubsException
+    {
+        /// <summary>
+        ///   Initializes a new instance of the <see cref="EventHubsCommunicationException"/> class.
```

Comment:
Forgot to edit: `EventHubsObjectClosedException`

Reply:
Same for the second constructor.

Reply:
d'oh... fixed.